src/main/java/org/kohsuke/github/Requester.java

```diff
@@ -169,6 +169,11 @@ public Requester with(@WillClose/*later*/ InputStream body) {
         return this;
     }
 
+	public Requester withNullable(String key, Object value) {
+		args.add(new Entry(key, value));
```

Comment:
according to this: https://developer.github.com/v3/repos/branches/#update-branch-protection, these values can be explicitly null and rather then change how `_with` worked, i created the new method.